### ISSUE TITLE
"Cannot find module 'simple-git'" error during CI deployment

### ISSUE THREAD

**Issue by richardvclam** [DataDog/serverless-plugin-datadog#218]
## Expected Behavior
Serverless CI running deployment should succeed.

## Actual Behavior
Serverless CI running deployment fails due to "Cannot find module 'simple-git'"

## Steps to Reproduce the Problem

  1. Install this library
  2. Push a change to github & let serverless CI pick up

## Specifications

  - Serverless Framework version: 2.72.0
  - Datadog Serverless Plugin version: 3.5.1
  - Lambda function runtime (Python 3.7, Node 10, etc.): Node 12
  
## Stacktrace
  
  ```
  Error: Cannot find module 'simple-git'
  Require stack:
  - /backend/node_modules/serverless-plugin-datadog/dist/src/git.js
  - /backend/node_modules/serverless-plugin-datadog/dist/src/index.js
  - /var/lang/lib/node_modules/serverless/lib/classes/PluginManager.js
  - /var/lang/lib/node_modules/serverless/lib/Serverless.js
  - /var/lang/lib/node_modules/serverless/scripts/serverless.js
  - /var/lang/lib/node_modules/serverless/bin/serverless.js
      at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
      at Function.Module._load (internal/modules/cjs/loader.js:667:27)
      at Module.require (internal/modules/cjs/loader.js:887:19)
      at require (internal/modules/cjs/helpers.js:74:18)
      at Object.<anonymous> (/backend/node_modules/serverless-plugin-datadog/dist/src/git.js:32:32)
      at Module._compile (internal/modules/cjs/loader.js:999:30)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
      at Module.load (internal/modules/cjs/loader.js:863:32)
      at Function.Module._load (internal/modules/cjs/loader.js:708:14)
      at Module.require (internal/modules/cjs/loader.js:887:19)
      at require (internal/modules/cjs/helpers.js:74:18)
      at Object.<anonymous> (/backend/node_modules/serverless-plugin-datadog/dist/src/index.js:41:15)
      at Module._compile (internal/modules/cjs/loader.js:999:30)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
      at Module.load (internal/modules/cjs/loader.js:863:32)
      at Function.Module._load (internal/modules/cjs/loader.js:708:14)
      at Module.require (internal/modules/cjs/loader.js:887:19)
      at require (internal/modules/cjs/helpers.js:74:18)
      at PluginManager.requireServicePlugin (/var/lang/lib/node_modules/serverless/lib/classes/PluginManager.js:164:14)
      at /var/lang/lib/node_modules/serverless/lib/classes/PluginManager.js:186:27
      at Array.map (<anonymous>)
      at PluginManager.resolveServicePlugins (/var/lang/lib/node_modules/serverless/lib/classes/PluginManager.js:183:10)
      at PluginManager.loadAllPlugins (/var/lang/lib/node_modules/serverless/lib/classes/PluginManager.js:139:10)
      at Serverless.init (/var/lang/lib/node_modules/serverless/lib/Serverless.js:213:30)
      at async /var/lang/lib/node_modules/serverless/scripts/serverless.js:589:7
  ```

## Workaround

My current workaround is to include `simple-git` in my project's package.json. This allows the CI to succeed

**Comment by IvanTopolcic** [comment]
Hi Richard, thanks for the report! This looks like something we overlooked in the latest release. I've created [this pr](https://github.com/DataDog/serverless-plugin-datadog/issues/218) to address the issue.

**Comment by IvanTopolcic** [comment]
Fixed in [3.5.2](https://github.com/DataDog/serverless-plugin-datadog/releases/tag/v3.5.2), as always, thank you so much for the report!
